<launch>

<arg name="sonarip" value="192.168.0.88"/>
<arg name="sonarport" value="5000"/>
<arg name="gpsip" value="_host:=127.0.0.1"/>
<arg name="gpsport" value="_port:=2947"/>
<arg name="loggerPath" value="/home/ubuntu/Poseidon/www/webroot/record/"/>
<arg name="portPath1" value="/dev/ttyACM0"/>


<node pkg="logger_binary" name="logger_binary" type="logger_binary_node" output="screen" args="$(arg loggerPath)"/>

<node pkg="logger_text" name="logger_text" type="logger_text_node" output="screen" args="$(arg loggerPath)"/>


<node pkg="sonar_nmea_0183_tcp_client" name="Sonar" type="sonar_nmea_0183_tcp_client_node" output="screen" args="$(arg sonarip) $(arg sonarport)"/>

<node pkg="imu_dummy" name="IMU" type="imu_dummy_node" output="screen"/>

<node pkg="gnss_dummy" name="GNSS" type="gnss_dummy_node" output="screen"/>


  


<!--<node pkg="catarob_control" name="catarob_control" type="catarob_control" output="screen"/>-->

<node pkg="catarob_control_websocket" name="catarob_control_websocket" type="catarob_control_websocket_node" args="$(arg loggerPath)"/>


<!--<node pkg="hydroball_data_websocket" name="hydroball_data_websocket" type="hydroball_data_websocket_node" args="$(arg loggerPath)"/>-->

<!--<node pkg="gnss_zed-f9p" name="gnss_zed_f9p" type="gnss_zed-f9p" output="screen" args="$(arg loggerPath1) $(arg portPath1)"/>-->


</launch>
